web/src/assets/locales/en.json

```diff
@@ -190,7 +190,7 @@
                 }
             }
         },
-        "build": {
+        "pipeline": {
```

Comment:
how can we rename this in weblate too?

so we dont have to retranslate it for all other langs?